.packit.yaml

```diff
@@ -55,6 +57,9 @@ jobs:
         # Even though the distro context is 8.6 we are setting the SYSTEM_RELEASE_ENV to 8.7
         # to tag the updated system correctly
       distros: [centos-8-latest, oraclelinux-8.6]
+    epel-9-x86_64:
+      # TBD: so far it's just guessing of the distro strings
+      distros: [oraclelinux-9-latest, AlmaLinux-OS-9.2, rocky-linux-9.2]
```

Comment:
We will need to get packit/packit-service#2072 resolved somehow, so the check identifiers for EL9 will make sense.
Also, we will call the AMIs from AWS directly in `tf_extra_params`